### PR TITLE
Updated "You will receive" Modal Design

### PR DESCRIPTION
**Pull request by bilalcorbacioglu**
You will receive while adding liq screen need redesign -> https://app.clickup.com/t/3buhm7y

### CODE REVIEW

File: src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx [hunk 1]
```diff
@@ -171,7 +171,7 @@ const ConfirmSwapDrawer: React.FC<Props> = (props) => {
   );
 
   const CardConfirmContent = (
-    <Box display="flex" flexDirection="column" p={10} height="100%">
+    <Box display="flex" flexDirection="column" px={20} pb={20} height="100%">
```
Comment (SarjuHansaliya) [inline src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx:174]: I think due to this padding, there is some mis alignment in drawer title and content. please check

Screenshot 2022-09-05 at 3 35 54 PM

File: src/components/Drawer/index.tsx [hunk 2]
```diff
@@ -16,15 +17,17 @@ export default function Drawer({ isOpen, onClose, children, title, backgroundCol
...
           </Text>
         </Box>
       )}
 
-      <Box position="absolute" right={10} top={10}>
-        <CloseIcon onClick={onClose} color={theme.text4} />
+      <Box position="absolute" right={20} top={20}>
+        <CloseCircle onClick={onClose}>
+          <X color={theme.primary} size={10} />
+        </CloseCircle>
```
Comment (SarjuHansaliya) [inline src/components/Drawer/index.tsx:30]: Close icon is not in center align with title. Please fix right & top position.

Screenshot 2022-09-05 at 3 35 54 PM

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/components/Pools/AddLiquidity/ConfirmPoolDrawer/styled.ts [hunk 3]
```diff
@@ -8,15 +8,14 @@ export const Root = styled(Box)`
...
 `;
 
 export const OutputText = styled(Text)`
   width: 100%;
-  font-style: italic;
-  font-size: 12px;
+  line-height: 18px;
   text-align: left;
-  color: ${({ theme }) => theme.text2};
+  color: ${({ theme }) => theme.text1};
```
[the remark this answers is not in the document]
Comment (bilalcorbacioglu) [reply src/components/Pools/AddLiquidity/ConfirmPoolDrawer/styled.ts]: Fixed

File: src/components/Drawer/styled.ts [hunk 4]
```diff
@@ -32,6 +31,12 @@ export const DrawerContent = styled.div`
   display: flex;
   flex-direction: column;
 `;
-export const CloseIcon = styled(X)<{ onClick: () => void }>`
+
+export const CloseCircle = styled.div<{ onClick: () => void }>`
   cursor: pointer;
+  background-color: rgba(255, 255, 255, 0.2);
```
[the remark this answers is not in the document]
Comment (bilalcorbacioglu) [reply src/components/Drawer/styled.ts]: fixed

File: src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx (hunk 1 above)
Comment (bilalcorbacioglu) [reply src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx:174]: This should be fine now, I fixed the conflicts.

File: src/components/Drawer/index.tsx (hunk 2 above)
Comment (bilalcorbacioglu) [reply src/components/Drawer/index.tsx]: This should be fine now, I fixed the conflicts.

**Review by SarjuHansaliya — APPROVED** [review]

**Pull request merged**